tests/integration/test_fim/test_files/test_max_eps/test_max_eps_synchronization.py

```diff
@@ -166,40 +181,43 @@ def test_max_eps_on_start(get_configuration, create_files, configure_environment
-                       (wazuh_conf_synchro.yaml) which includes configuration settings for
+                       (wazuh_sync_conf_max_eps.yaml) which includes configuration settings for
                        the 'wazuh-syscheckd' daemon and, these are combined with the
                        testing directories to be monitored defined in the module.
-
     expected_output:
         - r'.* Connected to the server .*'
         - r'.*Sending integrity control message'
 
     tags:
-        - realtime
         - scheduled
+        - realtime
+        - whodata
     '''
-    check_apply_test({'max_eps_synchronization'}, get_configuration['tags'])
     max_eps = int(get_configuration['metadata']['max_eps'])
 
     # Wait until the agent connects to the manager.
-    wazuh_log_monitor.start(timeout=90,
+    wazuh_log_monitor.start(timeout=TIMEOUT_CHECK_AGENT_CONNECT,
                             callback=callback_connection_message,
-                            error_message="Agent couldn't connect to server.").result()
+                            error_message=ERR_MSG_AGENT_DISCONNECT).result()
 
-    #  Find integrity start before attempting to read max_eps
-    wazuh_log_monitor.start(timeout=30,
+    # Find integrity start before attempting to read max_eps.
+    wazuh_log_monitor.start(timeout=TIMEOUT_CHECK_INTEGRATY_START,
                             callback=callback_integrity_message,
-                            error_message="Didn't receive integrity_check_global").result()
+                            error_message=ERR_MSG_INTEGRITY_CONTROL_MSG).result()
 
-    n_results = max_eps * 5
-    result = wazuh_log_monitor.start(timeout=120,
-                                     accum_results=n_results,
+    # Find integrity message for each file created after read max_eps.
+    total_file_created = max_eps + 5
+    result = wazuh_log_monitor.start(timeout=TIMEOUT_CHECK_EACH_INTEGRITY_MSG,
+                                     accum_results=total_file_created,
                                      callback=callback_integrity_message,
-                                     error_message=f'Received less results than expected ({n_results})').result()
+                                     error_message=f'Received less results than expected ({total_file_created})').result()
 
+    # Collect by time received the messages.
     counter = Counter([date_time for date_time, _ in result])
-    error_margin = (max_eps * 0.1)
 
-    for _, n_occurrences in counter.items():
-        assert n_occurrences <= round(
-            max_eps + error_margin), f'Sent {n_occurrences} but a maximum of {max_eps} was set'
+    # Check the number of occurrences of received messages by time.
+    for _, total_occurrences in counter.items():
+        assert total_occurrences <= max_eps, f'Sent {total_occurrences} but a maximum of {max_eps} was set'
+
+    # Delete all files created.
+    delete_path_recursively(test_directory)
```

Comment:
If some previous assert fails, this line will not be executed, so consider to add try-finally block in order to always delete the folder